src/Serializer/AbstractItemNormalizer.php

```diff
@@ -637,9 +645,8 @@ protected function getDataTransformer($data, string $to, array $context = []): ?
      * For a given resource, it returns an output representation if any
      * If not, the resource is returned.
      */
-    protected function transformOutput($object, array $context = [])
+    protected function transformOutput($object, string $outputClass, array $context = [])
```

Comment:
I'm thinking that to prevent BC change here perhaps it is better to add the parameter at the end with a default to null and to reinstate the call to `getOutputMethod` if it is null. I removed to prevent a duplicate call but failed to think about others who may be extending this class as it is not marked `@internal`

Reply:
:+1: indeed

Reply:
Thanks!